public/index.html

```diff
@@ -223,18 +223,14 @@ <h1 class="ready-to-start">Ready to start?</h1>
               <a href="http://www.codeminer42.com/">
                 <img class="footer-cm42-logo" src="./assets/images/codeminer42-logo-white.png" alt="codeminer logo">
               </a>
-
               <div class="footer-copyright">
-                <span>2017 Codeminer 42</span>
+                <span></span>
```

Comment:
Here I suggest to put the text inside a `<p>` element like this

```html
<div class="footer-copyright">
  <p>
    <span class="copyright-year"></span>
    Codeminer 42
   </p>
</div>
```

Reply:
And take care to HTML style not break. Hopefully not ;D

Reply:
broke not.
https://user-images.githubusercontent.com/392677/47670168-4e931800-db8b-11e8-971c-32eb5a3e5789.png
tks